docs/pages/build-reference/npm-cache-with-yarn.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Using npm cache with Yarn v1
```

Comment:
```suggestion
title: Using npm cache with Yarn Classic
```

Since Yarn refers to v1 as "Classic" I think we should stick to that name too:
* https://classic.yarnpkg.com/lang/en/

Reply:
![image](https://user-images.githubusercontent.com/852069/207360755-b2679911-2984-4bfb-99e0-de6f7ede1df1.png)
![image](https://user-images.githubusercontent.com/852069/207360829-916a0c8d-fdf6-4b3b-9e04-602913f514d1.png)

Left this on another PR not too long ago. These are the two variations that Yarn themselves uses. We went with Yarn 1 (Classic) in the other PR, but for a title, Yarn Classic is probably better, like you suggested.